tests/e2e/lib.sh

```diff
@@ -322,6 +322,8 @@ deploy_sensor() {
         # et al.
         kubectl -n "${sensor_namespace}" set resources deploy/sensor -c sensor --requests 'cpu=2' --limits 'cpu=4'
     fi
+    # Bump Sensor's memory to match operator installs.
+    kubectl -n "${sensor_namespace}" set resources deploy/sensor -c sensor --requests 'memory=4Gi' --limits 'memory=8Gi'
```

Comment:
I'd prefer to exclude this. When the operator changes the test deployment will be out of sync. The original CI flake is fixed by switching OSD to operator based install. Making a change that affects all cluster flavors and jobs is bound to cause other issues.
```suggestion
```